src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2372,4 +2431,34 @@ public function wp_cron_scheduled_check() {
 
 		set_transient( 'health-check-site-status-result', wp_json_encode( $site_status ) );
 	}
+
+	/**
+	 * Convert shorthand byte string to int byte
+	 *
+	 * E.g.
+	 *  "1G" (1 Gigabyte) = 1073741824
+	 *  "10M" (10 Megabytes) = 10485760
+	 *  "1K" (1 Kilobyte) = 1024
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param  string $size_string Shorthand byte string
+	 * @return int                 `$size_string` converted to bytes.
+	 */
+	public static function parse_ini_size( $size_string ) {
```

Comment:
This is a function that should probably live alongside [`size_format`](https://github.com/WordPress/wordpress-develop/blob/88c91eeaa6ab8d73bb3f5b301f9a0faf41e8e3e2/src/wp-includes/functions.php#L451) over in `functions.php`

Reply:
Yeah, I agree. Seems like a pretty universal function.